packages/components/app/styles/@hashicorp/design-system-components.scss

```diff
@@ -43,3 +43,17 @@
   clip: rect(1px, 1px, 1px, 1px) !important; /* 1 */
   clip-path: inset(50%) !important;  /* 2 */
 }
+
+// text alignment helpers - only use these if you absolutely must.
+
+.hds-text-left {
```

Comment:
[issue] these global classes are not ok. we should have "local" classes, specific only for the table

Reply:
I removed these.